### ISSUE TITLE
Version 7.0.0 results in security vulnerability and deprecation warnings

### ISSUE THREAD

**Issue by cduff** [NaturalNode/natural#736]
NPM install of natural@7.0.0 results in several security vulnerability and deprecation warnings. These did not occur for natural@6.12.0.

``` PowerShell
PS C:\Users\user\Dev\test> npm i natural@7.0.0
npm WARN deprecated loadash@1.0.0: Package is unsupport. Please use the lodash package instead.
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142

added 199 packages in 36s

13 packages are looking for funding
  run `npm fund` for details
PS C:\Users\user\Dev\test> npm audit
# npm audit report

request  *
Severity: moderate
Server-Side Request Forgery in Request - https://github.com/advisories/GHSA-p8p7-x288-28g6
Depends on vulnerable versions of tough-cookie
fix available via `npm audit fix --force`
Will install natural@6.12.0, which is a breaking change
node_modules/request
  lint  0.6.0 - 0.8.19
  Depends on vulnerable versions of request
  node_modules/lint
    natural  >=7.0.0
    Depends on vulnerable versions of lint
    node_modules/natural

tough-cookie  <4.1.3
Severity: moderate
tough-cookie Prototype Pollution vulnerability - https://github.com/advisories/GHSA-72xf-g2v4-qvf3
fix available via `npm audit fix --force`
Will install natural@6.12.0, which is a breaking change
node_modules/tough-cookie

4 moderate severity vulnerabilities

To address all issues (including breaking changes), run:
  npm audit fix --force
```

**Comment by Hugo-ter-Doest** [comment]
Thanks, I will take a look.

**Comment by ildella** [comment]
It is 'lint' causing it, due to old request dependencies. 
Lint should probably be a devDepencency

**Comment by Hugo-ter-Doest** [comment]
Thanks @ildella. The warnings are caused by lint:
```
$ npm install -D lint
npm WARN deprecated loadash@1.0.0: Package is unsupport. Please use the lodash package instead.
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
```
I moved lint to `devDependencies`. Warnings remain however.

**Comment by Hugo-ter-Doest** [comment]
With 7.0.1 we have an install without warnings:
```
$ npm install natural

added 57 packages, and audited 58 packages in 6s

5 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
```

**Comment by cduff** [comment]
Thanks for releasing 7.0.1 @Hugo-ter-Doest.

I tried upgrading to it and the security vulnerability and deprecation warnings are gone.

However, the next issue I'm facing is that 7.0.1 includes [@types/jasmine](https://www.npmjs.com/package/@types/jasmine) as a normal dependency which declares a bunch of global types (e.g. `expect`) that break my project compilation.

Shouldn't [@types/jasmine](https://www.npmjs.com/package/@types/jasmine) also be a dev dependency?

Let me know if you want me to create a separate issue for this.

**Comment by Hugo-ter-Doest** [comment]
Yes, this is a dev dependency. Will do so.

**Comment by Hugo-ter-Doest** [comment]
Fixed in #738

**Comment by cduff** [comment]
Great, working well for me again now @Hugo-ter-Doest. 👍
